modules/tribes/client/less/tribes-grid.less

```diff
@@ -7,7 +7,7 @@
     background-color: @brand-primary;
     background-size: cover;
     height: 250px;
-    width: 288px;
+    width: 100%;
```

Comment:
I wondered what this was about, but I understand now, as the react component ends up in a nested element, it doesn't get it's width set by the angular grid thing (which sets pixel widths directly on each element), so our inner element can just be full width :+1:

Reply:
@nicksellen Would you like to have a comment added there?

Reply:
Nope, seems fine without :)